### PR TITLE
Use filter_class for autodoc

### PR DESCRIPTION
**Pull request by TheoLechemia**
Not sur if we have to keep both : 
`get_schema_fields` of `GeotrekTouristicContentFilter` and `TouristicEventFilterSet`

### CODE REVIEW

File: geotrek/api/v2/filters.py [hunk 1]
```diff
@@ -483,6 +484,16 @@ def get_schema_fields(self, view):
         )
 
 
+class TouristicEventFilterSet(filters.FilterSet):
+    bookable = filters.BooleanFilter(
+        field_name=_("Bookable"), 
+        help_text=_("Filter events on bookable boolean : true/false exepected")
```
Comment (camillemonchicourt) [inline geotrek/api/v2/filters.py:490]: exepected >> expected

File: geotrek/tourism/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -374,3 +374,8 @@ msgstr "Valeur de géométrie invalide"
 
 msgid "Provider"
 msgstr "Fournisseur"
+
+
+msgid "Filter events on bookable boolean : true/false exepected"
+msgstr "Filtre les evenement sur le booléen 'bookable' : true/false attendu"
```
Comment (camillemonchicourt) [inline geotrek/tourism/locale/fr/LC_MESSAGES/django.po:380]: evenement >> événements

**Comment by Chatewgne** [comment]
Fix tests please :)

File: geotrek/api/v2/filters.py [hunk 3]
```diff
@@ -483,6 +484,16 @@ def get_schema_fields(self, view):
         )
 
 
+class TouristicEventFilterSet(filters.FilterSet):
+    bookable = filters.BooleanFilter(
+        field_name=_("Bookable"),
```
Comment (submarcos) [inline geotrek/api/v2/filters.py:489]:
```suggestion
        field_name="bookable",
```
this is field in database

Comment (submarcos) [inline geotrek/api/v2/filters.py:489]:
```suggestion
```
field_name is field in database. by default it's class atrribute

**Pull request merged**